com.jpexs.decompiler.flash.json

```diff
@@ -0,0 +1,58 @@
+{
+	"app-id": "com.jpexs.decompiler.flash",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "20.08",
+	"sdk": "org.freedesktop.Sdk",
+	"sdk-extensions": [
+		"org.freedesktop.Sdk.Extension.openjdk8"
+	],
+	"command": "ffdec.sh",
+	"finish-args": [
+		"--socket=x11",
+		"--env=PATH=/app/bin:/app/jre/bin:/usr/bin",
+		"--env=JAVA_HOME=/app/jre",
+		"--filesystem=host"
```

Comment:
why `host`?

Reply:
This is largely a personal preference. I sometimes want to access files outside of my home directory, and this software doesn't support portals. That said, I understand if home is preferable here, and I'm open to changing it.